### DISCUSSION TITLE
How to fixes this ,components comment double after submit post

### DISCUSSION THREAD

**Discussion by apriyansahjero** [livewire/livewire#5104] [1. Help]
![Screen Shot 2022-07-13 at 01 13 54](https://user-images.githubusercontent.com/25054806/178565818-c77af898-ff3e-41ef-92aa-99af9fe97f94.png)

**Comment by faramarzJafari** [comment]
Can i see your cods?

**Reply by apriyansahjero** [reply]
`<div>`
`Card post`
  `<div>`
         `@livewire('traits.post-comment', ['postId' => $post->id])  // comment form`
  `</div>`
` </div>`

**Reply by cnngraphics** [reply]
try to add this at the beginning of your function that handles the post-comment
```
public function commentHandlerFunction ( ){
    $this->reset( post-comment );
....
....
}
```
